src/diffusers/pipelines/pipeline_utils.py

```diff
@@ -301,9 +302,54 @@ def is_saveable_module(name, value):
 
             save_method(os.path.join(save_directory, pipeline_component_name), **save_kwargs)
 
+            if dduf_filename:
```

Comment:
Maybe a "TODO" on validating `dduf_filename` and that validation should probably go more at the beginning so that we can error out earlier?

Reply:
validation will be done from huggingface_hub side